airbyte-integrations/connectors/source-asana/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this should be a sample state, real stream name, column and some value

Reply:
Asana source doesn't support incremental sync for streams in this PR. So should I just delete this file?

Reply:
I just delete this file.